inc/Game.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <string>
+#include "../inc/Frame.hpp"
```

Comment:
you could add in CMakeLists.txt include_direcories(inc) and then you will be able to simply #include "Frame.hpp"